### ISSUE TITLE
Is the Bayesian posterior set available for the full set of >6,000 languages?

### ISSUE THREAD

**Issue by jamie-thompson** [rbouckaert/global-language-tree-pipeline#1]
I'd like to work with the all-languages trees generated for this paper.  Are they available?

**Comment by rbouckaert** [comment]
Yes, it is in the [release](https://github.com/rbouckaert/global-language-tree-pipeline/releases) area.
An update should be available soon (masa1300 is missing from the current set).
